docs/core/deploying/native-aot.md

```diff
@@ -0,0 +1,67 @@
+---
+title: NativeAOT deployment overview
+description: Learn what NativeAOT deployments are and why you should consider using it as part of the publishing your app with .NET 7 and later.
+author: lakshanf
+ms.author: lakshanf
+ms.date: 06/09/2022
+---
+# NativeAOT Deployment
```

Comment:
@samsp-msft Are we going to officially use "Native AOT" or "NativeAOT"?

Reply:
I'm ok with either and we have NativeAOT in our source (e.g. https://github.com/dotnet/runtime/pull/68950)

Reply:
NativeAOT IMO better for googling things. I would immidiately get only relevant results.

Reply:
@richlander @jkotas Do we care about how Native AOT/NativeAOT is spelled in docs? I recall there was some conversation about it but I can't find it now. The Preview 3 blog talks about "Native AOT".

Reply:
I think we should avoid this "ProjectN" style name. We should call it "Native AOT". We want to avoid the idea that we're creating a new brand or something similar. R2R is bad enough (on this topic). The more we can of this capability as a new build output, the better. I also don't think it always has to be capitalized, like "native AOT" should be fine within a sentence. It's "... native AOT differs from R2R since R2R is managed-first and purely an optimization on the normal functioning of the CoreCLR managed runtime."